src/battle_ai_util.c

```diff
@@ -554,48 +565,46 @@ s32 AI_CalcDamage(u32 move, u32 battlerAtk, u32 battlerDef, u8 *typeEffectivenes
             fixedBasePower = 0;
             break;
         }
-        normalDmg = CalculateMoveDamageVars(move, battlerAtk, battlerDef, moveType, fixedBasePower,
-                                             effectivenessMultiplier, weather, FALSE,
-                                             aiData->holdEffects[battlerAtk], aiData->holdEffects[battlerDef],
-                                             aiData->abilities[battlerAtk], aiData->abilities[battlerDef]);
 
         critChanceIndex = CalcCritChanceStageArgs(battlerAtk, battlerDef, move, FALSE, aiData->abilities[battlerAtk], aiData->abilities[battlerDef], aiData->holdEffects[battlerAtk]);
         if (critChanceIndex > 1) // Consider crit damage only if a move has at least +2 crit chance
```

Comment:
Looks good but could you make a config for people who want to turn off crit calcs at plus two or lower? 

Also unrelated to the config but right now I've noticed that if critChance is at +3 it will still calc as if it's the average. I think in this case it should be considered as it's always crit. Suggestion would be to check if `critChance  > 2` and assign -2  to it.